doc/nrf/releases/release-notes-2.1.3.rst

```diff
@@ -0,0 +1,129 @@
+
+* Updates the Matter ZAP Tool revision to be compatible with the :file:`zap` files that define the Data Model in |NCS| Matter samples.
+* Bluetooth® fixes:
+
+  * An issue in MPSL where the controller would assert when a Bluetooth role was running.
+    This was previously listed as the known issue with the issue ID <DRGN-17851>.
+  * An issue in MPSL where the controller would abandon a link, causing a disconnect on the remote side.
+    This was previously listed as the known issue with the issue ID <DRGN-18105>.
+  * An issue where RPA (random private address) was not changed when it should when BT_EXT_ADV was enabled.
+
+See :ref:`ncs_release_notes_213_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.1.3**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.1.3`_.
+
+IDE and tool support
+********************
+
+`nRF Connect for Visual Studio Code extension <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.1.3.
+SEGGER Embedded Studio Nordic Edition is no longer tested or recommended for new projects.
+
+:ref:`gs_app_tcm`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.1.3_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.1.3_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.1.3`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_213_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Bluetooth
+---------
+
+
```

Comment:
Empty section. We should duplicate entries from highlights, because highlights just summarize the salient entries from the changelog. Ideally, the highlights entries should be shorter than the changelog ones, so you can for example skip mentioning the line with the ID in the highlights and mention it only in the changelog.